nonstandardcode.py

```diff
@@ -1,16 +1,27 @@
-import numpy as np
```

Comment:
this file can be removed from the repository as the functionality is being provided by other modules.